src/models/mobileservices/pushvariantcr.js

```diff
@@ -229,6 +272,45 @@ export class PushVariantCR extends CustomResource {
               }
             }
           }
+        },
+        WEB_UPS_BINDING: {
+          comment: 'This is the set of rules that will be used to validate Web UPS Binding.',
+          executionConstraints: [
+            {
+              comment: "Execute this ruleset only when the field named 'CLIENT_TYPE' has value 'WEB'",
+              type: 'FIELD_VALUE',
+              name: 'CLIENT_TYPE',
+              value: 'Web'
+            }
+          ],
+          fields: {
+            platformConfig: {
+              alias: {
+                validation_rules: [
+                  {
+                    type: 'required',
+                    error: 'Web push requires an Alias'
+                  }
+                ]
+              },
+              publicKey: {
```

Comment:
I believe we could add a validator to validate that the passed in object are really a public and a private key

Reply:
This we can do. How do we validate the public and private keys, as in what format should we be checking for.